kuma/settings/common.py

```diff
@@ -1092,19 +1073,19 @@ def JINJA_CONFIG():
 BASKET_URL = 'https://basket.mozilla.com'
 BASKET_APPS_NEWSLETTER = 'app-dev'
 
-KUMASCRIPT_URL_TEMPLATE = 'http://developer.mozilla.org:9080/docs/{path}'
+KUMASCRIPT_URL_TEMPLATE = 'http://localhost:9080/docs/{path}'
```

Comment:
It's the same in all environments, we can wrap it in a `config` call when needed (e.g. when we separate the web heads from the kumascript servers). This is me being careful about adding configuration variables.